src/docGenerator.ts

```diff
@@ -54,8 +54,8 @@ export class DocGenerator {
   }
 
   async generateSpecification(patternsSchema: any) {
+    const disabledDefaultPatternIds = ['MD013','MD043','MD041','MD009','MD040','MD031','MD047'];
```

Comment:
Preferably move this into its own function so it has better visibility and if the logic of how we choose default changes in the future the change will be centralized. Something like:
```typescript
isDefaultPattern(patternId: string): boolean {
  const disabled = ['MD013','MD043','MD041','MD009','MD040','MD031','MD047']
  
  return !disabled.includes(patternId)
}
```